tests/transforms/test_compose.py

```diff
@@ -59,18 +61,20 @@ def test_compose_n2v(ordered_array):
     array = ordered_array((2, 5, 5))
 
     # transform lists
-    mean, std = 0.5, 0.5
+    means, stds = compute_normalization_stats(image=array)
+
 
     transform_list_pydantic = [
-        NormalizeModel(mean=mean, std=std),
-        XYFlipModel(seed=seed),
+        NormalizeModel(image_means=means, image_stds=stds),
+        NDFlipModel(seed=seed),
```

Comment:
`XYFlipModel` is the new name, I must have missed that in the merge 😞 

This must trigger a failure in the tests!